db/migrate/20160203195736_remove_docker_image_schema.rb

```diff
@@ -0,0 +1,39 @@
+class RemoveDockerImageSchema < ActiveRecord::Migration
```

Comment:
I'm not sure if all the steps can happen here. For example, there are some runcible calls and an action invocation

https://github.com/Katello/katello/pull/5744